core/modules/modulesconf.py

```diff
@@ -0,0 +1,46 @@
+from viur.core.bones import StringBone
+from viur.core.tasks import StartupTask
+from viur.core import conf, db
+from viur.core.i18n import translate
+from viur.core.skeleton import Skeleton, SkeletonInstance
+from viur.core.prototypes import List, BasicApplication
+
+
+class ModulesConfSkel(Skeleton):
+    kindName = "viur-modules-conf"
+    name = StringBone(descr=translate("modulename"), readOnly=True)
+    help_text = StringBone(descr=translate("module helptext"))
+    help_text_add = StringBone(descr=translate("add helptext"))
+    help_text_edit = StringBone(descr=translate("edit helptext"))
```

Comment:
Can you add some description (tooltip) / comments what's the purpose of the bones? Are they for the vi?

Reply:
Yes, these are additional tooltips in the administration. For example, general data maintenance notes or a general description of the module.